playbooks/roles/edx_django_service/tasks/main.yml

```
@@ -212,6 +212,16 @@
     - install
     - install:app-requirements
 
+- name: install Open edX Atlas translation tool
+  get_url:
+    url: "https://github.com/openedx/openedx-atlas/releases/download/v0.1.1/atlas"
```

Does it make sense to need to manually bump the version here, or should we try to always use latest?

```suggestion
    url: "https://github.com/openedx/openedx-atlas/releases/latest/download/atlas"
```

---

I think this wouldn't be a good idea.

What happens when we break the `atlas` interface in version e.g. 2.1.x?

I think it would break open-release branches `configuration` and would require a patch from platform operators.

One way to manage this, is to publish atlas as both `pip` and `npm` package which would let us install it as `v1.x` instead of `v1.0.0` or `latest`.

---

I think we can agree that this issue can work both ways. So if you're strongly leaning towards `latest` I can amend it.

---

Yeah, I guess with the limitations on directly downloading from GH releases we're best off using a specific tagged version. It'd probably be good to have a place to keep track of all the places we're downloading atlas from releases so we can make a bunch of version bump PRs when needed.